### PR TITLE
Added cawbird profile

### PR DESCRIPTION
**Pull request by kortewegdevries**
I think I needed to rebase it, Cawbird's final commit is df8c39a

### CODE REVIEW

File: etc/profile-a-l/cawbird.profile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+apparmor
+caps.drop all
+machine-id
+netfilter
+nodvd
+no3d
+nogroups
+nonewprivs
+noroot
+nosound
```
Comment (rusty-snake) [inline etc/profile-a-l/cawbird.profile:29]: Can it play videos?
---
Yes, please rebase.

Comment (kortewegdevries) [reply etc/profile-a-l/cawbird.profile:29]: I've tested it with two gifs and one video from WHO,I don't get a lot of videos on my timeline. No sound though, do we need sound and hyperlinking? I'll close this for now and rebase and open another PR later if that's okay.

Comment (rusty-snake) [reply etc/profile-a-l/cawbird.profile:29]: > I'll close this for now and rebase and open another PR later if that's okay.

You can also force push.

> do we need sound

If sound works w/o FJ and breaks with, we will get new issues.

Comment (kortewegdevries) [reply etc/profile-a-l/cawbird.profile:29]: And hyperlinking? Cawbird has some API restrictions,like polls/group DMs for example,not that I use those...

Comment (rusty-snake) [reply etc/profile-a-l/cawbird.profile:29]: If it is not such uncommon, we maybe should add a note which can be uncommented.

**Pull request closed without merging**